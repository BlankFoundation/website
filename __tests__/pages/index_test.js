// This is a test of the BlankIndex component.
// It is the top level component so it is rendered as the index page.
import React from 'react';
import TestRenderer from 'react-test-renderer';
import Index from '../../pages/index';

describe("Index", () => {
  it('renders correctly', () => {
    const rendered = TestRenderer.create(<Index />).toJSON();
    expect(rendered).toMatchSnapshot()
  });
});