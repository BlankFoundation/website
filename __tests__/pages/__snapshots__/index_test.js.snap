// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
<div>
  <div
    className="font-roboto-mono text-sm bg-white text-gray-900 flex flex-col h-screen"
  >
    <div
      className="py-12 p-6 bg-white text-black leading-10"
    >
      <div
        className="px-24 flex flex-row justify-between"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Blank.Foundation
        </a>
        <div
          className="space-x-2 flex flex-row justify-between"
        >
          Music
        </div>
      </div>
    </div>
    <div
      className="flex items-center justify-center flex-grow"
    >
      <div
        className="inline-block"
      >
        <p>
          CONTENT!
        </p>
        <p>
          <a
            className="text-blue-500 underline"
            href="/mint"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Mint!
          </a>
        </p>
      </div>
    </div>
    <div
      className=" p-6 bg-gray-900 text-white text-xs leading-10"
    >
      <div
        className="px-24 flex flex-row justify-between"
      >
        <div
          className="space-x-4 flex flex-row justify-start"
        >
          <a
            href="https://discord.gg/HZM2jcHdEU"
            rel="noopener noreferrer"
            target="_blank"
          >
            Discord
          </a>
          <a
            href="https://twitter.com/blankfdn"
            rel="noopener noreferrer"
            target="_blank"
          >
            Twitter
          </a>
          <a
            href="https://medium.com/@blankfdn"
            rel="noopener noreferrer"
            target="_blank"
          >
            Medium
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
